sp800-63d/vot_mapping.md

```diff
@@ -118,25 +118,87 @@ FAL maps into the VoT component A using a numeric identifier. Only one numeric i
 
 #### Assertion Presentation
 
-In addition, the presentation mechanism MAY be indicated using an alphabetic identifier. Multiple aphabetic identifiers MAY be used simultaneously. The presentation mechanism SHALL be sufficient or greater than that which is required by the FAL value indicated in the vector.
-
 |Presentation|Component Value|
 |:----:|:--:|
 |Front channel|Af|
 |Back channel|Ab|
 |Additional features implemented beyond requirements for asserted FAL|Ax|
 
-Any discrepancy in the assessed and implemented assurance level SHOULD be documented in a digital identity acceptance statement, per SP 800-63-3 Section 5.5.
+## How to Use the Values in Vectors
 
-### Authenticator Management to VoT
+The implemented IAL, AAL, and/or FAL component value(s) SHALL be indicated in the vector. Only one numeric identifier SHALL be used for each xAL. Any discrepancies in the assessed and implemented assurance level SHOULD be documented in a digital identity acceptance statement, per SP 800-63-3 Section 5.5.
```

Comment:
While this covers the combination requirements, I'm a little wary of it being only found down here. Maybe more examples in each section would help?